draft-ietf-quic-transport.md

```diff
@@ -1296,6 +1291,22 @@ connection ID, it should expect packets on the new connection ID to start at 18.
 A packet with the new connection ID and a packet number of 17 is discarded as
 being in error.
 
+#### Packet Number Gap
+
+In order to avoid linkage, the packet number gap MUST be externally
+indistinguishable from random. The packet number gap for a connection
+ID with sequence number is computed by encoding the sequence number
+as a 32-bit integer in big-endian format, and then computing:
+
+~~~
+Gap = HKDF-Expand-Label(packet_number_secret,
+                        "QUIC packet sequence gap", sequence, 4)
```

Comment:
"keyed hash" isn't really a well-formed cryptographic construct. What you want here is a PRF, and these days that typically means using HMAC (which is what HKDF is under the covers). So, if you really prefer, we could inline the HMAC expansion of HKDF-Expand-Label, but I'm not really seeing what the benefit of that is in this instance, given that, as Martin says, if we move packet protection into this document, we'll need HKDF anyway.

Jana, it might help me if it were a bit clearer about what you were optimizing for here. Any implementation of this document is going to need HKDF (and even if you were to swap out TLS, it's quite likely you would end up using HKDF), so it's not really implementation complexity, so is it just that you don't like having a citation to some construct exposed by TLS? If so, that seems like something we can deal with much later in the document cycle if we still feel that way.